### PR TITLE
Move slashing protection cli options into a mixin

### PR DESCRIPTION
**Pull request by jframe**
Move slashing protection cli options into a mixin to cleanup config code.

### CODE REVIEW

**Review by usmansaleem — APPROVED** [review]
LGTM

File: core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java [hunk 1]
```diff
@@ -73,31 +74,22 @@
...
     this.azureKeyVaultParameters = azureKeyVaultParameters;
   }
 
   private Optional<SlashingProtection> createSlashingProtection(
-      final boolean slashingProtectionEnabled,
-      final String slashingProtectionDbUrl,
-      final String slashingProtectionDbUser,
-      final String slashingProtectionDbPassword) {
-    if (slashingProtectionEnabled) {
+      SlashingProtectionParameters slashingProtectionParameters) {
```
Comment (rain-on) [inline core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java:85]: final?

**Review by rain-on — APPROVED** [review]
missed a final I THINK

File: core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java (hunk 1 above)
Comment (jframe) [reply core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java]: done

**Pull request merged**